Sources/StreamChatUI/ChatMessageList/ChatThreadVC.swift

```diff
@@ -468,16 +401,7 @@ open class _ChatThreadVC<ExtraData: ExtraDataTypes>:
     }
 
     open func messageContentViewDidTapOnThread(_ indexPath: IndexPath?) {
-        guard let indexPath = indexPath else { return log.error("IndexPath is not available") }
-        guard let channel = channelController.channel else { return }
-
-        let controller = _ChatThreadVC<ExtraData>()
-        controller.channelController = channelController
-        controller.messageController = channelController.client.messageController(
-            cid: channel.cid,
-            messageId: messageController.replies[indexPath.item].id
-        )
-        navigationController?.show(controller, sender: self)
+        log.error("Nestead threads are not supported")
     }
```

Comment:
If this is not supported why do we have the method here?

Reply:
we don't provide empty default implementations for `ChatMessageContentViewDelegate`. should we?